app/blueprints/assessments/templates/macros/fund_dashboard_summary.html

```diff
@@ -48,90 +76,76 @@ <h3 class="govuk-body govuk_summary-card__title govuk-!-font-size-24">{{summary.
-                <a class="govuk-link" href="{{ summary.feedback_export_href }}">
-                    Export applicants feedback information
-                </a>
-            {% endif %}
 
-        {% endif %}
-        {% if summary.assessment_stats %}
-            {{ govukTable({
-            "caption": "",
-            "firstCellIsHeader": false,
-            "rows": [
-            [{'html': '<strong>Assessment closing date</strong>', 'classes': 'govuk-!-width-one-third'},
-            {'text': summary.assessment_stats.date | datetime_format("%d %B %Y at %H:%M") }],
-            [{'html': '<strong>Applications received</strong>'},
-            {'text': summary.assessment_stats.total_received}],
-            [{'html': '<strong>Assessments completed</strong>'},
-            {'text': summary.assessment_stats.completed}],
-            [{'html': '<strong>Assessments started</strong>'},
-            {'text': summary.assessment_stats.started}],
-            [{'html': '<strong>QA Complete</strong>'},
-            {'text': summary.assessment_stats.qa_complete}],
-            [{'html': '<strong>Stopped assessments</strong>'},
-            {'text': summary.assessment_stats.stopped}],
-            ],
-            }) }}
+        {% elif summary.status.has_assessment_opened %}
 
-            <a class="govuk-link" data-qa="dashboard_summary" href="{{ summary.assessments_href }}">
-                View all {% if summary.is_assessment_active_status %}active{% else %}
-                closed{% endif %} assessments
-            </a>
-            {{ round_links(summary.access_controller, summary.round_application_fields_download_available, summary.export_href, summary.feedback_export_href, summary.assessment_tracker_href) }}
+            {{ govukTable({
+                "caption": "",
+                "firstCellIsHeader": false,
+                "rows": [
+                    [{'html': '<strong>Assessment closing date</strong>', 'classes': 'govuk-!-width-one-third'},
+                    {'text': summary.assessment_stats.date | datetime_format("%d %B %Y at %H:%M") }],
+                    [{'html': '<strong>Applications received</strong>'},
+                    {'text': summary.assessment_stats.total_received}],
+                    [{'html': '<strong>Assessments completed</strong>'},
+                    {'text': summary.assessment_stats.completed}],
+                    [{'html': '<strong>Assessments started</strong>'},
+                    {'text': summary.assessment_stats.started}],
+                    [{'html': '<strong>QA Complete</strong>'},
+                    {'text': summary.assessment_stats.qa_complete}],
+                    [{'html': '<strong>Stopped assessments</strong>'},
+                    {'text': summary.assessment_stats.stopped}],
+                    ],
+                })
+            }}
 
         {% endif %}
+        {% if not summary.status.is_application_not_yet_open %}
```

Comment:
Can you double-check the logic here? We are not supposed to show 'assessment tracker export' link for live rounds I think
![image](https://github.com/communitiesuk/funding-service-design-assessment/assets/127315890/d27618c1-faf2-4bc8-9b60-dd2dacb552f8)

Reply:
I think we also hide  `export applicant information` link on Live round as there are no applications in the assess store it might crash.